src/js/actions/policy.js

```diff
@@ -189,6 +213,61 @@ define(function (require, exports) {
     removePointerPolicies.writes = [locks.PS_APP, locks.JS_POLICY];
 
     /**
+     * Temporarily disable keyboard policies, caching the previous state
+     *
+     * @return {Promise}
+     */
+    var disableKeyboardPolicies = function () {
+        var policyStore = this.flux.store("policy"),
+            kind = PolicyStore.eventKind.KEYBOARD;
+
+        _cachedKeyboardPolicies = policyStore.getPolicies(kind);
+
+        return adapterUI.getKeyboardPropagationMode()
+            .bind(this)
+            .then(function (mode) {
+                _cachedKeyboardMode = mode;
+                policyStore.clearPolicies(kind);
+            }).then(function () {
+                return _syncPolicies.call(this, kind);
+            })
+            .then(function () {
+                return adapterUI.setKeyboardPropagationMode(
+                    { defaultMode: adapterUI.keyboardPropagationMode.FOCUS_PROPAGATE });
+            });
+    };
+    disableKeyboardPolicies.reads = [];
+    disableKeyboardPolicies.writes = [locks.PS_APP, locks.JS_POLICY];
+
+    /**
+     * Re-enabled cached keyboard policies
+     *
+     * @return {Promise}
+     */
+    var reenabledKeyboardPolicies = function () {
```

Comment:
Actions are verbs, so I think this should be called `reenableKeyboardPolicies`! That would also match `disableKeyboardPolicies` better.